ui/src/components/common/Entity.jsx

```diff
@@ -34,8 +34,6 @@ class EntityLabel extends Component {
         <span className='EntityLabel untitled'>
           {icon && <Schema.Icon schema={schema}/>}
           {icon && ' '}
-          <FormattedMessage id='entity.label.missing'
-                            defaultMessage="Untitled" />
```

Comment:
No! Don't remove it everywhere! Just don't show it in the sidebar when the entity has no name, but this would make all untitled entities everywhere un-clickable.